interface/tags_filters/start.php

```diff
@@ -85,5 +85,97 @@ function update_tags_filters()
     $plugin->migrate();
 }
 add_action( 'update_plugin', 'update_tags_filters' );
+}
+
+if ( $GLOBALS['facility_acl']==1 ) {
+///////////////////////////////////////////////////////////////////////////////
+////test add of facility acl stuff.
+function get_facilities_to_show( $username )
+{
+    // User facility is stored in users_facility table as facility_id
+    $sql = "SELECT UF.facility_id, U.facility_id AS default_facility
+        FROM users U
+        JOIN users_facility UF
+        ON UF.table_id = U.id
+        WHERE
+            U.username = ? AND
+            UF.tablename = ?";
```

Comment:
@aethelwulffe @teryhill I think a `;` is not required for a single query statement inside string. I think I haven't seen it inside string anywhere in code base.